### PR TITLE
[ESS][8.14] Documentation for Value list modal

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Contributes to https://github.com/elastic/security-docs/issues/5052

Main value list docs that need to be reviewed:
- [Manage value lists](https://security-docs_bk_5195.docs-preview.app.elstc.co/guide/en/security/master/value-lists-exceptions.html#manage-value-lists) - Minor change, added an intro sentence 
- [Edit value lists](https://security-docs_bk_5195.docs-preview.app.elstc.co/guide/en/security/master/value-lists-exceptions.html#edit-value-lists) - New section that's solely dedicated to updating value lists
- [Export or remove value lists](https://security-docs_bk_5195.docs-preview.app.elstc.co/guide/en/security/master/value-lists-exceptions.html#export-remove-value-lists) - Slightly revised to focus on how to export and delete value lists
- [Import rules API](https://security-docs_bk_5195.docs-preview.app.elstc.co/guide/en/security/master/rules-api-import.html) - Updated the third note in the intro
- [Export rule API](https://security-docs_bk_5195.docs-preview.app.elstc.co/guide/en/security/master/rules-api-export.html) - Update the note in the intro

I also refreshed related docs and screenshots to account for the renamed **Import value list** button (the new name is **Manage value lists**). These pages would benefit from a quick spot check:
- [About detection rules](https://security-docs_bk_5195.docs-preview.app.elstc.co/guide/en/security/master/about-rules.html)
- [Install and manage Elastic prebuilt rules](https://security-docs_bk_5195.docs-preview.app.elstc.co/guide/en/security/master/prebuilt-rules-management.html)
- [Manage detection rules](https://security-docs_bk_5195.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-management.html)
- [Create a detection rule](https://security-docs_bk_5195.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-create.html)
- [Elastic Security UI](https://security-docs_bk_5195.docs-preview.app.elstc.co/guide/en/security/master/es-ui-overview.html)
- [Monitor and troubleshoot rule executions](https://security-docs_bk_5195.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-monitor.html)

Twin Serverless PR: https://github.com/elastic/staging-serverless-security-docs/pull/322

### CODE REVIEW

File: docs/detections/value-list-exceptions.asciidoc [hunk 1]
```diff
@@ -37,27 +37,54 @@ act as delimiters.
...
-the new file are appended to the previously imported values.
+NOTE: If you import a file with a name that already exists, a new list is not created. The imported values are added to the existing list instead.
 
-[[edit-value-lists]]
+[[manage-value-lists]]
 [discrete]
 === Manage value lists
 
-To view, delete, or export existing value lists:
+You can edit, remove, or export existing value lists.
```
Comment (nastasha-solomon) [inline docs/detections/value-list-exceptions.asciidoc:55]: Update all of the icons on this page.

Comment (nastasha-solomon) [reply docs/detections/value-list-exceptions.asciidoc:55]: Resolved in [735dff1](https://github.com/elastic/security-docs/pull/5195/commits/735dff16af04cc810bbb71283cb0dbd65054115e).

File: docs/detections/value-list-exceptions.asciidoc [hunk 2]
```diff
@@ -37,27 +37,54 @@ act as delimiters.
...
++
+You can also filter by the `updated_by` (for example, `updated_by:testuser`), or `updated at` (for example, `updated_at < now`) fields.  
+** **Add individual items to the list**: Click **Create list item**, enter a value, then click **Add list item**.
+** **Bulk upload list items**: Drag or select the `csv` or `txt` file that contains the values that you want to add, then click **Upload**.
+** **Edit a value**: In the Value column, go to the value you want to edit and click the **Edit** button (image:images/edit-value-list-item.png[Edit button from Manage value lists window,15,15]). When you're done editing, click the **Save** button (image:images/save-value-list-item-changes.png[Save button from Manage value lists window,15,15]) to save your changes. Click the **Cancel** button (image:images/cancel-value-list-item-changes.png[Cancel button from Manage value lists window,15,15]) to revert your changes.
```
Comment (nastasha-solomon) [inline docs/detections/value-list-exceptions.asciidoc:71]:
```suggestion
** **Edit a value**: In the Value column, go to the value you want to edit and click the **Edit** button (image:images/edit-value-list-item.png[Edit button from Manage value lists window,15,15]). When you're done editing, click the **Save** button (image:images/save-value-list-item-changes.png[Save button from Manage value lists window,18,18]) to save your changes. Click the **Cancel** button (image:images/cancel-value-list-item-changes.png[Cancel button from Manage value lists window,18,18]) to revert your changes. 
```

File: docs/detections/value-list-exceptions.asciidoc [hunk 3]
```diff
@@ -37,27 +37,54 @@ act as delimiters.
...
+==== Edit value lists
+
+. Go to **Rules** → **Detection rules (SIEM)**.
+. Click **Manage value lists**. The **Manage value lists** window opens. 
+. In the **Value lists** table, click the value list you want to edit.
+. Do any of the following:
+
+** **Filter items in the list**: Use the KQL search bar to find values in the list. Depending on your list's type, you can filter by the `keyword`, `ip_range`, `ip`, or `text` fields. For example, to filter by Gmail addresses in a value list of the `keyword` type, enter `keyword:*gmail.com` into the search bar. 
++
+You can also filter by the `updated_by` (for example, `updated_by:testuser`), or `updated at` (for example, `updated_at < now`) fields.
```
Comment (benironside) [inline docs/detections/value-list-exceptions.asciidoc:68]:
```suggestion
You can also filter by the `updated_by` field (for example, `updated_by:testuser`), or the `updated at` field (for example, `updated_at < now`).  
```
Had to re-read the sentence a couple times to get it. I think this scans more easily

File: docs/detections/value-list-exceptions.asciidoc [hunk 4]
```diff
@@ -37,27 +37,54 @@ act as delimiters.
...
++
+You can also filter by the `updated_by` (for example, `updated_by:testuser`), or `updated at` (for example, `updated_at < now`) fields.  
+** **Add individual items to the list**: Click **Create list item**, enter a value, then click **Add list item**.
+** **Bulk upload list items**: Drag or select the `csv` or `txt` file that contains the values that you want to add, then click **Upload**.
+** **Edit a value**: In the Value column, go to the value you want to edit and click the **Edit** button (image:images/edit-value-list-item.png[Edit button from Manage value lists window,15,15]). When you're done editing, click the **Save** button (image:images/save-value-list-item-changes.png[Save button from Manage value lists window,18,18]) to save your changes. Click the **Cancel** button (image:images/cancel-value-list-item-changes.png[Cancel button from Manage value lists window,18,18]) to revert your changes. 
+** **Remove a value**: Click the **Remove value list** button (image:images/remove-value-list-item.png[Remove value list button from Manage value lists window,15,15]) to delete a value from the list.
```
Comment (benironside) [inline docs/detections/value-list-exceptions.asciidoc:72]:
```suggestion
** **Remove a value**: Click the **Remove value** button (image:images/remove-value-list-item.png[Remove value list button from Manage value lists window,15,15]) to delete a value from the list.
```
Not familiar with the 8.14 UI but I thought this is just for removing a single value, not a list.

**Review by benironside — COMMENTED** [review]
Looks good! Left a couple suggestions, hope they're helpful

**Review by benironside — APPROVED** [review]

**Comment by nastasha-solomon** [comment]
I'll merge this once I've confirmed that the new copy is available in 8.14-BC5.

**Comment by nastasha-solomon** [comment]
Checked siem.dev server and confirmed that the new copy is available in the UI.

**Pull request merged**